endpoint/endpoint.go

```diff
@@ -201,10 +206,14 @@ func New(config Config) (*Endpoint, error) {
 		conn, _ := grpc.Dial(config.Address, dialOptions...)
 		client = data_receiver.NewDataReceiverServiceClient(conn)
 	}
+	regDialOption := grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{}))
```

Comment:
Shouldn't this client setup:
1. Be included in the else{} above?
2. Use the dial options with the interceptors similar to data_receiver along with the TLS?

Reply:
Updated